index.js

```diff
@@ -38,5 +41,7 @@ function nanotiming (name) {
 }
 
 function noop (cb) {
-  if (cb) onIdle(cb)
+  if (cb) onIdle(function () {
+    cb(new Error('performance api unavailable'))
```

Comment:
could you prefix this with `nanotiming: performance API unavailable`? Thanks!

Reply:
To clarify: adding a prefix to an error (especially coming from frameworks) makes it so much easier to track down; can't trust all folks to have proper source maps, and debug tooling enabled in all environments. This makes it a little easier (: